### PR TITLE
Jerry´s pull request

### PR DESCRIPTION
**Pull request by GerardoDel9**
Method names changed, in the last build version some method names started with capital

### CODE REVIEW

File: TestingPro/src/com/deloitte/test/amazon/page/home/AmazonSignIn.java [hunk 1]
```diff
@@ -12,13 +12,13 @@
 	public AmazonSignIn(WebDriver driver) {
 		this.driver = driver;
 	}
-	public void InputEmail(String email) {
+	public void inputEmail(String email) {
 		driver.findElement(By.xpath("//input[@id='ap_email']")).sendKeys(email);
```
Comment (navi911407) [inline TestingPro/src/com/deloitte/test/amazon/page/home/AmazonSignIn.java]: WebElemenst declaration move out from the methods and paste it below or above public AmazonSignIn(WebDriver driver) and should be done like this @FindBy(xpath="//input[@id='advancedFinder']")
	private WebElement advanceFinderToogle;

File: TestingPro/src/com/deloitte/test/amazon/page/home/AmazonSignIn.java [hunk 2]
```diff
@@ -12,13 +12,13 @@
...
-	public void InputEmail(String email) {
+	public void inputEmail(String email) {
 		driver.findElement(By.xpath("//input[@id='ap_email']")).sendKeys(email);
 	}
-	public void ClickContinue() {
+	public void clickContinue() {
 		driver.findElement(By.xpath("//input[@id='continue']")).click();
 	}
-	public void InputPassword(String pass) {
+	public void inputPassword(String pass) {
```
Comment (navi911407) [inline TestingPro/src/com/deloitte/test/amazon/page/home/AmazonSignIn.java]: Methods that receive parameters must be private

File: TestingPro/src/com/deloitte/test/amazon/page/home/AmazonSignIn.java [hunk 3]
```diff
@@ -31,7 +31,7 @@ public void getFail() {
 	public void clickCreateAccount() {
 		driver.findElement(By.xpath("//a[@id='createAccountSubmit']	")).click();
 	}
-	public void RegisterForm(int noElement,String Data)
+	public void registerForm(int noElement,String Data)
```
Comment (navi911407) [inline TestingPro/src/com/deloitte/test/amazon/page/home/AmazonSignIn.java]: Methods that receive parameters must be private

File: TestingPro/src/com/deloitte/test/amazon/tests/Test1Amazon.java [hunk 4]
```diff
@@ -21,13 +21,13 @@ public static void main (String [] args) throws InterruptedException {
 		
 		
 		amaHome.clickSignIn();
-		amaSign.InputEmail("proyecto.bk201@gmail.com");
-		amaSign.ClickContinue();
-		amaSign.InputPassword("proyecto.bk201");
+		amaSign.inputEmail("proyecto.bk201@gmail.com");
```
Comment (navi911407) [inline TestingPro/src/com/deloitte/test/amazon/tests/Test1Amazon.java]: User and password must not be hardcoded

File: TestingPro/src/com/deloitte/test/amazon/tests/Test1Amazon.java [hunk 5]
```diff
@@ -21,13 +21,13 @@ public static void main (String [] args) throws InterruptedException {
 		
 		
 		amaHome.clickSignIn();
-		amaSign.InputEmail("proyecto.bk201@gmail.com");
-		amaSign.ClickContinue();
-		amaSign.InputPassword("proyecto.bk201");
+		amaSign.inputEmail("proyecto.bk201@gmail.com");
+		amaSign.clickContinue();
+		amaSign.inputPassword("proyecto.bk201");
```
Comment (navi911407) [inline TestingPro/src/com/deloitte/test/amazon/tests/Test1Amazon.java]: User and password must not be hardcoded

**Review by navi911407 — CHANGES_REQUESTED** [review]
Please modify comments made

**Review by GerardoDel9 — COMMENTED** [review]
Fixed

**Pull request merged**